spaces/unicore/index.json

```
@@ -0,0 +1,27 @@
+{
+  "token": "0x5506861bbb104Baa8d8575e88E22084627B192D8",
+  "name": "UniCore",
+  "network": "1",
+  "symbol": "UNICORE",
+  "skin": "unicore",
+  "domain": "vote.uvault.finance",
+  "strategies": [
+    {
+      "name": "erc20-balance-of",
+      "params": {
+        "address": "0x5506861bbb104Baa8d8575e88E22084627B192D8",
+        "symbol": "UNICORE",
+        "decimals": 18,
```

Please just remove comma and we are done! 
```suggestion
        "decimals": 18
```